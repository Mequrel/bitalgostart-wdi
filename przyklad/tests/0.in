10
9 8 11 20 3 1 9 5 9 10