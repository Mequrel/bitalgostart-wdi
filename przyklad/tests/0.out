1 3 5 8 9 9 9 10 11 20